bin/MadGraph5_aMCatNLO/cards/production/13TeV/WPrime_tb/WPrime_tb_proc_card.dat

```diff
@@ -0,0 +1,17 @@
+import model WPrimetb_UFO
+define bs = b b~
+define p = p bs
+define j = j bs
+define wp = wp+ wp-
+define ls = l+ l- vl vl~
+define ws = w+ w-
+define ts = t t~
+generate p p > wp, (wp > ts bs, (ts > ws bs, ws > ls ls) )
+add process p p > wp j, (wp > ts bs, (ts > ws bs, ws > ls ls) )
+add process p p > wp, (wp > ts bs j, (ts > ws bs, ws > ls ls) )
```

Comment:
Does this syntax make sense?
Can wprime decay to top bottom and a jet?

Reply:
Similar comments to below other syntaxes

Reply:
The syntax in the last line is to simulate hard final state gluon radiation. I've attached a tarball containing the diagrams produced.
The rest of the mistakes seem to have been from using a template a little too lazily and have been corrected.

Thank you for the quick and thorough feedback.

[diagrams.tar.gz](https://github.com/cms-sw/genproductions/files/5788648/diagrams.tar.gz)

Reply:
So it seems you don't want any offshell effects here, am I correct?
p p > wp > ts bs
p p > wp > ts bs j
p p > wp > ts bs j j
would give you the same diagrams but also include offshell Wprime contributions as well.

Reply:
I've just updated the process card to include offshell contributions.

Reply:
This is what I had for myself.
It's "p p > wr > n1 lep" process where mwr >> mn1 = 100 GeV. So it's a bit similar to what you have (" p p > wp > ts bs", mts ~ 200 GeV).

Keep in mind that by adding the offshell contribution you "might" get low mass tail bump (look at the brown upper triangle) which seems irregular but true in physics due to PDF effects. What you had before will give you something similar to the purple line. The width will depend on the nature of wp in your model so I'm not sure how low it will reach.

But looking at the card generating script, if mwp = 1100GeV is your maximal mass point the effect will not be that much big. For my case mwr = 3000 and mn1 = 100 GeV was where the offshell effect became visible.

[ios_scan_3128783726.pdf](https://github.com/cms-sw/genproductions/files/5811820/ios_scan_3128783726.pdf)